azurerm/config.go

```diff
@@ -228,6 +229,9 @@ type ArmClient struct {
 	managementGroupsClient             managementgroups.Client
 	managementGroupsSubscriptionClient managementgroups.SubscriptionsClient
 
+	// Media
+	mediaServicesClient media.MediaservicesClient
```

Comment:
since this is a new Resource Provider - could we add `microsoft.media` (case-sensitive) to the list of Resource Providers which Terraform will register: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/required_resource_providers.go#L44